mkvm.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |spec|
   spec.name          = "mkvm"
   spec.version       = MKVM::VERSION
   spec.authors       = ["CoverMyMeds"]
-  spec.email         = ["smerrill@covermymeds.com", "dsajner@covermymeds.com", "dmorris@covermymeds.com"]
+  spec.email         = ["smerrill@covermymeds.com", "dsajner@covermymeds.com", "nchowning@covermymeds.com"]
```

Comment:
Don't think we need to email Dan anymore

Reply:
Good catch

Reply:
Fixed